syntax/text/parser3/src/Luna/Pass/Parsing/Parser.hs

```diff
@@ -305,22 +306,38 @@ buildIR = \(Spanned cs ast) -> addCodeSpan cs =<< case ast of
                     | otherwise         -> Nothing
                 _ -> Nothing
             realNumber =
-                let dot = tok == Ast.Operator Name.acc
-                    isNumber ast = case Ast.unspan ast of
+                let isNumber ast = case Ast.unspan ast of
                         Ast.Number _ -> True
                         _            -> False
                     lIsNum = isNumber l
                     rIsNum = isNumber r
                 in dot && lIsNum && rIsNum
 
+            -- luna/luna#301
```

Comment:
I'd like to see some more description in the codebase than just the issue number. This will be deprecated by the new syntax, I assume so perhaps something about that.